app/src/main/java/com/example/tasky/feature_agenda/data/mapper/AgendaMapper.kt

```diff
@@ -111,27 +111,33 @@ fun AttendeeEntity.toAttendee(): Attendee {
     )
 }
 
-fun PhotoEntity.toPhoto(): Photo {
-    return Photo(
+fun PhotoEntity.toPhoto(): EventPhoto {
+    return EventPhoto.Local(
         key = key,
-        url = url
+        uri = uri
     )
 }
```

Comment:
You're not supposed to save the local photos. As per requirements, the app should disallow adding photos in offline-mode (everything else would add too much complexity to finish it in 10 weeks). Therefore, you can just save the remote photos with their URLs in Room